commit/plugin.go

```diff
@@ -56,7 +56,7 @@ type Plugin struct {
 func NewPlugin(
 	_ context.Context,
 	donID plugintypes.DonID,
-	nodeID commontypes.OracleID,
+	oracleID commontypes.OracleID,
 	oracleIDToP2pID map[commontypes.OracleID]libocrtypes.PeerID,
```

Comment:
This has been bothering me for a while, can this be renamed to `oracleIDToP2PID`?

Reply:
yeah, we also don't want to provide it twice, it already in the cfg